stacks/index.js

```diff
@@ -9,7 +9,7 @@ export default function (app) {
   app.setDefaultFunctionProps({
     runtime: 'nodejs16.x',
     environment: {
-      NODE_OPTIONS: "--enable-source-maps",
+      NODE_OPTIONS: "--enable-source-maps  --experimental-fetch",
```

Comment:
needed so that we can use `aggregate-client` with ucanto client without passing a fetch implementation manually over the place